compiler/rustc_resolve/src/late.rs

```diff
@@ -1266,15 +1266,15 @@ impl<'ra: 'ast, 'ast, 'tcx> Visitor<'ast> for LateResolutionVisitor<'_, 'ast, 'r
 
     fn visit_where_predicate(&mut self, p: &'ast WherePredicate) {
         debug!("visit_where_predicate {:?}", p);
-        let previous_value = replace(&mut self.diag_metadata.current_where_predicate, Some(p));
+        let previous_value = replace(&mut self.diag_metadata.current_where_predicate, Some(&p));
```

Comment:
Unnecessary change?

Reply:
Oh, yes, it's probably accidentally introduced by the rebasing work.